scripts/operators-lifecycle/rhoam-operator-logs/rhoam-operator-logs.sh

```diff
@@ -0,0 +1,95 @@
+  for value in $(oc get po -n redhat-rhoam-3scale | grep apicast-staging | grep Running| awk '{print $1}'); do oc logs $value -c envoy-sidecar -n redhat-rhoam-3scale > $value.log ; done
+
+  # backend-cron
+  for value in $(oc get po -n redhat-rhoam-3scale | grep backend-cron | grep Running| awk '{print $1}'); do oc logs $value -c backend-cron -n redhat-rhoam-3scale > $value.log ; done
+  # backend-listener
+  for value in $(oc get po -n redhat-rhoam-3scale | grep backend-listener | grep Running| awk '{print $1}'); do oc logs $value -c backend-listener -n redhat-rhoam-3scale > $value.log ; done
+  # backend-worker
+  for value in $(oc get po -n redhat-rhoam-3scale | grep backend-worker | grep Running| awk '{print $1}'); do oc logs $value -c backend-worker -n redhat-rhoam-3scale > $value.log ; done
+
+  # system-app
+  for value in $(oc get po -n redhat-rhoam-3scale | grep system-app | grep Running| awk '{print $1}'); do oc logs $value -c system-master -n redhat-rhoam-3scale > $value-system-master.log ; done
+  for value in $(oc get po -n redhat-rhoam-3scale | grep system-app | grep Running| awk '{print $1}'); do oc logs $value -c system-provider -n redhat-rhoam-3scale > $value-system-provider.log ; done
+  for value in $(oc get po -n redhat-rhoam-3scale | grep system-app | grep Running| awk '{print $1}'); do oc logs $value -c system-developer -n redhat-rhoam-3scale > $value-system-developer.log ; done
+  # system-memcache
+  for value in $(oc get po -n redhat-rhoam-3scale | grep system-memcache | grep Running| awk '{print $1}'); do oc logs $value -c memcache -n redhat-rhoam-3scale > $value.log ; done
+  # system-sidekiq
+  for value in $(oc get po -n redhat-rhoam-3scale | grep system-sidekiq | grep Running| awk '{print $1}'); do oc logs $value -c system-sidekiq -n redhat-rhoam-3scale > $value.log ; done
+  # system-sphinx
+  for value in $(oc get po -n redhat-rhoam-3scale | grep system-sphinx | grep Running| awk '{print $1}'); do oc logs $value -c system-sphinx -n redhat-rhoam-3scale > $value.log ; done
+
+  # zync
+  for value in $(oc get po -n redhat-rhoam-3scale | grep zync | grep Running| awk '{print $1}'); do oc logs $value -n redhat-rhoam-3scale > $value.log ; done
+
+  # marin3r
+  for value in $(oc get po -n redhat-rhoam-3scale | grep marin3r | grep Running| awk '{print $1}'); do oc logs $value -n redhat-rhoam-3scale > $value.log ; done
+
+  cd ../..
+}
+
+function keycloak_logs(){
+  create_directory "keycloak"
+  cd "$main_log_dir/keycloak"
+
+  for value in $(oc get po -n redhat-rhoam-rhsso | grep keycloak | grep Running| awk '{print $1}'); do oc logs $value -c keycloak -n redhat-rhoam-rhsso > $value-rhsso.log ; done
+  for value in $(oc get po -n redhat-rhoam-user-sso | grep keycloak | grep Running| awk '{print $1}'); do oc logs $value -c keycloak -n redhat-rhoam-user-sso > $value-user-sso.log ; done
+
+  cd ../..
+}
+
+function tar_logdir(){
+    tar -zcf - ./"$main_log_dir"
+}
+
+3scale_logs
+keycloak_logs
```

Comment:
We are missing the 3scale-operator, rhsso-operator, user-sso-operator , marin3r-operator, and rhoam-operator